replication-api-impl/src/main/java/com/connexta/replication/api/impl/persistence/spring/FilterIndexRepository.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.replication.api.impl.persistence.spring;
+
+import com.connexta.replication.api.impl.persistence.pojo.FilterIndexPojo;
+import java.util.Optional;
+import org.springframework.data.repository.CrudRepository;
+
+public interface FilterIndexRepository extends CrudRepository<FilterIndexPojo, String> {
```

Comment:
I will let Patrick add a comment about javadoc here :)

Reply:
Why? You are doing good :-D